t/web/term_table/nz.rb

```diff
@@ -17,4 +17,13 @@
       subject.css('#term h1').text.must_include '51st Parliament'
     end
   end
+
+  describe 'HTML validation' do
+    before { get '/new-zealand/house/term-table/51.html' }
+
+    it 'has no errors in the term-table page' do
+      skip if `which tidy`.empty?
```

Comment:
[Again](https://github.com/everypolitician/viewer-sinatra/pull/15566), I don't think we want to duplicate this in every test like this.